src/lib/layouts/DocsArticle.svelte

```diff
@@ -16,11 +16,14 @@
     import { scrollToTop } from '$lib/actions/scrollToTop';
 
     import { Feedback } from '$lib/components';
+    import { page } from '$app/stores';
 
     export let title: string;
     export let toc: Array<TocItem>;
     export let back: string | undefined = undefined;
     export let date: string | undefined = undefined;
+
+    $: isQuickStartArticle = $page.route.id?.includes('/docs/quick-starts/');
```

Comment:
This seems quite hacky since it doesn't catch pages that aren't quickstarts. For example, [migrations](https://appwrite.io/docs/advanced/migrations/firebase):

<img width="1278" alt="image" src="https://github.com/user-attachments/assets/058b385c-30fc-4e85-bd5d-dd377f2884b0">

or [AI](https://appwrite.io/docs/products/ai/integrations/perplexity):

<img width="1275" alt="image" src="https://github.com/user-attachments/assets/264a0317-2450-4cc2-9d6b-bdff8d39efdc">


I think we need a better solution to handle all cases where we're using this numbered step.

Reply:
Makes sense. I'll improve the logic for this. Although we could have a completely different component with content with sections but that feels like an overkill and the docs [.md] might require a good amount of refactors.

Reply:
Done. It now only resizes if section with numbering is used on content. Let me know if this works now.